spark-3.1-spanner-lib/src/main/java/com/google/cloud/spark/spanner/SpannerUtils.java

```diff
@@ -330,156 +316,154 @@ public static Integer toSparkDate(com.google.cloud.Date dc) {
     return zonedDateTimeToSparkDate(zdt);
   }
 
-  public static GenericArrayData timestampIterToSpark(Iterable<Timestamp> tsIt) {
-    List<Long> dest = new ArrayList<>();
-    tsIt.forEach((ts) -> dest.add(toSparkTimestamp(ts)));
-    return new GenericArrayData(dest.toArray(new Long[0]));
-  }
-
-  public static GenericArrayData zonedDateTimeIterToSparkDates(Iterable<ZonedDateTime> tsIt) {
-    List<Integer> dest = new ArrayList<>();
-    tsIt.forEach((ts) -> dest.add(zonedDateTimeToSparkDate(ts)));
-    return new GenericArrayData(dest.toArray(new Integer[0]));
-  }
+  public static void convertRowAt(
```

Comment:
Where is the unit test for this method?

Reply:
There is no direct unit test for this method (like the original `spannerStructToInternalRow`).

Reply:
That's the problem...